Functions/xASL_quant_HadamardDecoding.m

```diff
@@ -101,7 +101,12 @@
                                  1  1 -1 -1  1  1 -1 -1;
                                  1 -1  1 -1  1 -1  1 -1];
 		end    
-	end
+    end
+    
+    if strcmp(xQ.Vendor,'Philips')
```

Comment:
x.Q.Vendor

Reply:
That was correct. We only pass x.Q as input, not the whole x, so inside this function. It is called xQ ;)